packages/-ember-data/package.json

```diff
@@ -31,63 +31,64 @@
     "@ember-data/store": "4.5.0-alpha.4",
     "@ember/edition-utils": "^1.2.0",
     "@ember/string": "^3.0.0",
-    "@embroider/macros": "^1.2.0",
+    "@embroider/macros": "^1.8.3",
     "@glimmer/env": "^0.1.7",
     "broccoli-merge-trees": "^4.2.0",
-    "ember-auto-import": "^2.2.4",
+    "ember-auto-import": "^2.4.2",
     "ember-cli-babel": "^7.26.11",
-    "ember-cli-typescript": "^5.0.0",
-    "ember-inflector": "^4.0.1"
+    "ember-cli-typescript": "^5.1.0",
+    "ember-inflector": "^4.0.2"
   },
   "devDependencies": {
-    "@babel/plugin-transform-typescript": "^7.16.8",
+    "@babel/plugin-transform-typescript": "^7.18.8",
     "@ember-data/unpublished-test-infra": "4.5.0-alpha.4",
     "@ember/optional-features": "^2.0.0",
-    "@ember/test-helpers": "^2.6.0",
-    "@glimmer/component": "^1.0.0",
+    "@ember/test-helpers": "~2.7.0",
+    "@glimmer/component": "^1.1.2",
     "@types/ember": "^4.0.0",
     "@types/ember-qunit": "^5.0.0",
     "@types/ember-testing-helpers": "^0.0.4",
-    "@types/ember__debug": "^4.0.0",
-    "@types/ember__test-helpers": "^2.6.1",
+    "@types/ember__debug": "^4.0.1",
+    "@types/ember__test-helpers": "^2.8.0",
     "@types/ember__utils": "^4.0.0",
     "@types/rsvp": "^4.0.4",
-    "broccoli-concat": "^4.2.4",
+    "broccoli-concat": "^4.2.5",
     "broccoli-stew": "^3.0.0",
     "broccoli-string-replace": "^0.1.2",
     "broccoli-test-helper": "^2.0.0",
     "broccoli-uglify-sourcemap": "^4.0.0",
-    "ember-cli": "~4.3.0",
+    "ember-cli": "~4.5.0",
     "ember-cli-app-version": "^5.0.0",
     "ember-cli-dependency-checker": "^3.3.1",
-    "ember-cli-htmlbars": "^6.0.1",
-    "ember-cli-inject-live-reload": "^2.0.2",
+    "ember-cli-htmlbars": "^6.1.0",
+    "ember-cli-inject-live-reload": "^2.1.0",
     "ember-cli-sri": "^2.1.1",
-    "ember-cli-terser": "~4.0.1",
+    "ember-cli-terser": "~4.0.2",
     "ember-cli-test-loader": "^3.0.0",
     "ember-cli-typescript-blueprints": "^3.0.0",
     "ember-cli-yuidoc": "^0.9.1",
-    "ember-compatibility-helpers": "^1.2.0",
+    "ember-compatibility-helpers": "^1.2.6",
     "ember-decorators-polyfill": "^1.1.5",
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-export-application-global": "^2.0.1",
-    "ember-load-initializers": "^2.1.1",
+    "ember-load-initializers": "^2.1.2",
     "ember-maybe-import-regenerator": "^1.0.0",
     "ember-qunit": "^5.1.5",
     "ember-resolver": "^8.0.3",
-    "ember-source": "~4.3.0",
+    "ember-source": "~4.5.0",
     "ember-source-channel-url": "^3.0.0",
     "ember-try": "^2.0.0",
     "json-typescript": "^1.1.2",
     "loader.js": "^4.7.0",
-    "pretender": "^3.4.3",
-    "qunit": "^2.17.0",
+    "pretender": "^3.4.7",
+    "qunit": "^2.19.1",
     "qunit-dom": "^2.0.0",
-    "typescript": "~4.6.2",
-    "webpack": "^5.37.1"
+    "typescript": "~4.7.4",
+    "webpack": "^5.73.0"
   },
   "engines": {
-    "node": "12.* || >= 14.*"
+    "node": "12.* || >= 14.*",
+    "yarn": "1.22.19"
```

Comment:
this change is causing issues for people installing ember-data with older yarn versions:

> ember-data@4.6.0: The engine "yarn" is incompatible with this module. Expected version "1.22.19". Got "1.22.18"

Reply:
Ah. I forgot that is going to get applied externally. We should remove it then. Annoying cause it's nice for folks that don't use Volta.